lib/jsdom/browser/Window.js

```diff
@@ -93,12 +94,11 @@ function Window(options) {
     }
   });
   // https://html.spec.whatwg.org/#session-history
-  this._sessionHistory = [{
+  this._sessionHistory = options.sessionHistory || new SessionHistory({
```

Comment:
Well, I would leave out the option for now, until we actually have something that uses it.